src/cocoa/toga_cocoa/dialogs.py

```diff
@@ -132,4 +132,19 @@ def open_file(window, title, file_types, multiselect):
         filename_or_filenames = (paths if multiselect else
                                  panel.URL.path)
         return filename_or_filenames
+
+
+def select_folder(window, title):
+    dialog = NSOpenPanel.alloc().init()
```

Comment:
At column 14: (F405) 'NSOpenPanel' may be undefined, or defined from star imports: .libs